pkgs/applications/gis/openorienteering-mapper/default.nix

```diff
@@ -81,7 +89,8 @@ mkDerivation rec {
       and provides a free alternative to the existing proprietary solution.
     '';
     homepage = "https://www.openorienteering.org/apps/mapper/";
-    license = licenses.gpl3;
+    changelog = "${src.meta.homepage}/releases/tag/v${version}";
```

Comment:
Can we use the full link here? Otherwise you can't easily open it in your browser from this file.

Reply:
You can't open the link as-is anyway (version should be filled). Generally, someone would evaluate that field from some scripting:
```sh
$ nix-instantiate --eval -A openorienteering-mapper.meta.changelog
"https://github.com/OpenOrienteering/mapper/releases/tag/v0.9.5"
```

Reply:
Yeah true but when you do not use src.meta.homepage I can just click to the github repo and then find the releases tab.

Reply:
Well, fixed.